.buildkite/scripts/install_deps.sh

```diff
@@ -4,6 +4,27 @@ source .buildkite/scripts/tooling.sh
 
 set -euo pipefail
 
+platform_type="$(uname)"
+hw_type="$(uname -m)"
+platform_type_lowercase="${platform_type,,}"
```

Comment:
Nit. It looks a bit unexpected to me to call platform_type to the operating system. And `platform_type` is never used, so maybe we can reserve this name to the final form that we use.
```suggestion
uname_os_type="$(uname)"
hardware_name="$(uname -m)"
os_type="${uname_os_type,,}"
```